music_album.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'item'
+
+class MusicAlbum < Item
+  attr_reader :on_spotify, :genre, :source, :label
+
+  def initialize(on_spotify, genre, publish_date, source, label)
+    super(publish_date)
+    @on_spotify = on_spotify
+    @genre = genre
+    @source = source
+    @label = label
+  end
+
+  def can_be_archived?
+    return true if super && @on_spotify == true
```

Comment:
- [ ] **[Optional]** Dear @SabiMantock, The condition `@on_spotify == true` in the `can_be_archived?` method can be simplified to just `@on_spotify`, as `@on_spotify` is expected to be a boolean value.